config.m4

```diff
@@ -440,11 +471,21 @@ if test "$PHP_MONGODB" != "no"; then
     fi
   fi
   if test "$PHP_LIBBSON" = "no"; then
-    m4_include(src/libbson/build/autotools/CheckAtomics.m4)
-    m4_include(src/libbson/build/autotools/FindDependencies.m4)
-    m4_include(src/libbson/build/autotools/m4/ac_compile_check_sizeof.m4)
-    m4_include(src/libbson/build/autotools/m4/ac_create_stdint_h.m4)
-    AC_CREATE_STDINT_H([$srcdir/src/libbson/src/bson/bson-stdint.h])
+    sinclude(src/libbson/build/autotools/CheckAtomics.m4)
```

Comment:
Side question: once we `sinclude()` a file, is an include within that file relative to the file's own path? I wonder if we could clean up a lot of this by having a single gateway include that then includes all of these general m4 files.